### ISSUE TITLE
Names fail to map, then succeed on a second attempt

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/opentree#779]
For the Nexus file that is the attachment to https://groups.google.com/d/msg/opentreeoflife/Cj1tw2jWnAc/Zgpr6wpQCAAJ , a large number of names (Brian says 1/3 of them) fail to map on first attempt, and then succeed on a second attempt. (All of the names in that file come from the taxonomy itself.)

When I tried it, three names that failed and then succeeded were Polyalthia xanthopetala, Polyalthia lateriflora, and Polyalthia stenophylla, but all the ones I tried had this property.

**Comment by jar398** [comment]
Of course it's conceivable this is a TNRS issue, not a curator issue.

**Comment by jimallman** [comment]
I'm having trouble reproducing this issue. The names I've tried (in the suggested file) consistently map or don't; the ones that fail have obviously tricky labels like these:
  - Campylobacter jejuni (species ncbi:197 in family Campylobacteraceae)
  - Terminalia sp. B1 KGD-2012

I did not have any trouble mapping the names from Polyalthia above. I do see a problem with some names like 'Solms-laubachia baiogoinensis', which fail in OTU mapping but work as a taxon search in the synthetic-tree viewer. Perhaps the old hyphen problem again? Checking this now...
